crates/starknet-server/tests/common/mod.rs

```diff
@@ -1,53 +1,78 @@
-use std::net::TcpListener;
-use std::process::{Child, Command, Stdio};
-use std::{thread, time};
-
-use hyper::client::HttpConnector;
-use hyper::{Client, StatusCode, Uri};
-use thiserror::Error;
-
-#[derive(Error, Debug)]
-pub enum TestError {
-    #[error("Cannot start Devnet")]
-    DevnetNotStartable,
-    #[error("No free ports")]
-    NoFreePorts,
+#[cfg(test)]
+pub mod constants {
+    pub const HOST: &str = "localhost";
+    pub const MIN_PORT: u16 = 1025;
+    pub const MAX_PORT: u16 = 65_535;
+    pub const SEED: usize = 42;
+    pub const ACCOUNTS: usize = 3;
+
+    pub const MAX_RETRIES: usize = 10;
```

Comment:
comment about MAX_RETRIES would be nice

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/64/commits/b2fecb5fc1f3956b7a53e6f0375db96291b8c72f